src/Api/Model/Shared/ProjectModel.php

```diff
@@ -189,8 +189,6 @@ public function remove()
      */
     public function addUser($userId, $role)
     {
-        ProjectModelMongoMapper::instance();
-        //        $ProjectModelMongoMapper::mongoID($userId)
```

Comment:
you can remove this.

Reply:
I only meant to get rid of the comment line, sorry about that.

Reply:
So, in this PR, both lines were taken out. Chris said that the instance() call isn't doing anything. From my testing it seems like it's OK to remove it

Reply:
yeah, sorry for the confusion, I shouldn't said anything, just wasn't paying close attention 🤪